erpnext/patches/v14_0/single_to_multi_dunning.py

```diff
@@ -7,7 +7,15 @@ def execute():
 	frappe.reload_doc("accounts", "doctype", "overdue_payment")
 	frappe.reload_doc("accounts", "doctype", "dunning")
 
-	all_dunnings = frappe.get_all("Dunning", filters={"docstatus": ("!=", 2)}, pluck="name")
+	filters = {"docstatus": ("!=", 2)}
```

Comment:
Shouldn't this be ?
```suggestion
	filters = {"docstatus": ("==", 1)}
```

Reply:
No, draft Dunnings also need to be updated with the payment schedule.

But line 52 could be: `if dunning.status != "Resolved" and dunning.docstatus == 1`